lib/trino-filesystem-s3/src/test/java/io/trino/filesystem/s3/TestS3FileSystemConfig.java

```diff
@@ -70,6 +77,11 @@ public void testExplicitPropertyMappings()
                 .put("s3.streaming.part-size", "42MB")
                 .put("s3.requester-pays", "true")
                 .put("s3.max-connections", "42")
+                .put("s3.connection-ttl", "1m")
```

Comment:
Use different values for each one so we can be sure they are mapped correctly

Reply:
Done